Added a new waybarlayout and improved some hyprland config files

ALEX5402 opened the pull request:
# Pull Request

## Description
- Added a new waybar layout here is a image of that layout 
![image](https://github.com/prasanthrangan/hyprdots/assets/76860596/637b9835-4927-4219-a087-d422aa2062db)

## Type of change
-  Added a key bind for choosing a color from current screen using hyprpricker
-  Added set display to maximum resolution and maximum refresh rate
-  Add  a new waybar layout
-  Added some good features on fish config

Enigma1309 commented on Configs/.config/fish/config.fish:
```diff
@@ -4,12 +4,28 @@ if status is-interactive
+export LC_ALL=en_US.UTF-8
+
+
+# Cleanup local orphaned packages
+function cleanup
+    while pacman -Qdtq
+        sudo pacman -R (pacman -Qdtq)
+    end
+end
```
this may cause the removal of wanted packages
in my case packages like these were also listed
![image](https://github.com/prasanthrangan/hyprdots/assets/94917588/db353195-5d54-4739-aab3-12f6a42d8056)

ALEX5402 replied:
Ok then i should make this feature as optional and i will comment it on fish config

ALEX5402 replied:
> this may cause the removal of wanted packages in my case packages like these were also listed ![image](https://private-user-images.githubusercontent.com/94917588/313487481-db353195-5d54-4739-aab3-12f6a42d8056.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4NjYzNzEsIm5iZiI6MTcxMDg2NjA3MSwicGF0aCI6Ii85NDkxNzU4OC8zMTM0ODc0ODEtZGIzNTMxOTUtNWQ1NC00NzM5LWFhYjMtMTJmNmE0MmQ4MDU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE5VDE2MzQzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3ZjQ3OTQ2NWJmM2ZkOWZlYmU5YTAxNDYxMTZkOWUzOGQ0Njc0MDk1NWI5NDIxN2ExNDJiMjU3N2YwOWU0MWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RvOlpZXB-yA3Tj46w0O0YMx2QNIR4oRkIUuh2D6V3Mk)

about this issue please check [this](https://github.com/ALEX5402/hyprdots/commit/c43a7e040a9d25dff0f60538527c1dca2d5c1a32#diff-f63351d384d0e6c52749b4f21daa510d5f613dc0b941493d03833d84824263c1) commit

Enigma1309 commented on Configs/.config/hypr/hyprland.conf:
```diff
@@ -16,6 +16,8 @@
 # See https://wiki.hyprland.org/Configuring/Monitors/
 
 monitor = ,preferred,auto,auto
+monitor=,highrr,auto,auto #for maximum refreashrate
+monitor=,highres,auto,auto # for max resoloution
```
i think these lines should be commented out by default

Enigma1309 commented on Configs/.config/hypr/hyprland.conf:
```diff
@@ -71,11 +60,14 @@ input {
-
-    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
-    force_no_accel = 1
+    numlock_by_default = false # numbpad toggle
+    sensitivity = 0.2 # -1.0 - 1.0, 0 means no modification.
+    force_no_accel = false
+    accel_profile = "adaptive" #flat
+    natural_scroll = false
```
This is a pretty nice change, since a lot of user create issue about this

Enigma1309 commented on Configs/.config/waybar/config.jsonc:
```diff
@@ -203,9 +224,9 @@
-        "on-click-middle": "~/.config/hyprdots/scripts/volumecontrol.sh -o m",
-        "on-scroll-up": "~/.config/hyprdots/scripts/volumecontrol.sh -o i",
-        "on-scroll-down": "~/.config/hyprdots/scripts/volumecontrol.sh -o d",
+        "on-click-middle": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -o m",
+        "on-scroll-up": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -o i",
+        "on-scroll-down": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -o d",
```
You did a oops here, this was better before, idk everyone here is alex :D

ALEX5402 replied:
ok i will change it i forgot to see there

Enigma1309 commented on Configs/.config/waybar/config.jsonc:
```diff
@@ -225,19 +246,102 @@
-        "on-click-middle": "~/.config/hyprdots/scripts/volumecontrol.sh -i m",
-        "on-scroll-up": "~/.config/hyprdots/scripts/volumecontrol.sh -i i",
-        "on-scroll-down": "~/.config/hyprdots/scripts/volumecontrol.sh -i d",
+        "on-click-middle": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -i m",
+        "on-scroll-up": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -i i",
+        "on-scroll-down": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -i d",
```
same mistake  here

ALEX5402 replied:
> same mistake here

please check [this](https://github.com/ALEX5402/hyprdots/commit/c43a7e040a9d25dff0f60538527c1dca2d5c1a32
) commit that mistake was fixed now

kRHYME7 commented:
Hey I think you've pushed your whole configuration on the PR.

prasanthrangan commented:
Hi, thanks for your contribution!
There are some conflicts here, so please try to keep the PR focused on one change, so that we can avoid issues, conflicts or rework.

ALEX5402 commented:
> Hi, thanks for your contribution! There are some conflicts here, so please try to keep the PR focused on one change, so that we can avoid issues, conflicts or rework.

ok please checkout [this](https://github.com/ALEX5402/hyprdots/commit/dffa172eca1884e1670caaa4f6e7bb028988cacc) commit i fixed the conflicts in here and added some new window rules

ALEX5402 commented:
> Hi, thanks for your contribution! There are some conflicts here, so please try to keep the PR focused on one change, so that we can avoid issues, conflicts or rework.
> Hi, thanks for your contribution! There are some conflicts here, so please try to keep the PR focused on one change, so that we can avoid issues, conflicts or rework.

 Please check out [this](https://github.com/ALEX5402/hyprdots/commit/dffa172eca1884e1670caaa4f6e7bb028988cacc) commit, i fixed the conflicts in here and added some new window rules
